validate/static/stage.css

```
@@ -0,0 +1,409 @@
+  margin: 3px 0 0 !important; max-width: none !important;
+  font-size: 12.5px; line-height: 1.5; color: var(--doc-muted);
+}
+.fstep-note { margin-top: 7px !important; color: var(--doc-faint); }
+.fdrop {
+  display: grid; grid-template-columns: 92px 1fr; gap: 12px; align-items: baseline;
+  padding: 9px 16px 9px 0; margin-left: 16px;
+  border-left: 2px dashed var(--border-2);
+}
+.fdrop-count {
+  text-align: right; font-variant-numeric: tabular-nums;
+  font: 600 13.5px/1.5 ui-monospace, SFMono-Regular, Menlo, monospace;
```

Stylelint (17.14.0)

[error] Overridden property "font-variant-numeric" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)